doc/README.md

```diff
@@ -0,0 +1,22 @@
+# DME-Core documentation
+This is the "ConfigurationObject" for the entire DME.
+
+It reads all the configuration files, the Operating System version and the Window version and apply a desired configuration accordingly.
+
+## Requirements
+[Detailed requirement list here](00DMERequirements.md)
+
+## Design
+[DME Design is available here](01DMEDesign.md)
+
+## DME Config File Protocol
+[DME Configuration File Protocol details available here](02DMEConfigFileP.txt)
+
+## DME Theme Protocol
+The DME Theme Protocol is inspired by Rich Goodwin and Chir (Little Yellow Moon) works.
```

Comment:
```suggestion
The DME Theme Protocol is inspired by Rich Goodwin's and Chir's (Little Yellow Moon) works.
```
 Altereatively `The DME Theme Protocol is inspired by the works of Rich Goodwin's and Chir's (Little Yellow Moon).`